src/modes.rs

```diff
@@ -65,7 +65,7 @@ pub fn context(args: Cli) {
                 "{}/{}:{}",
                 &DEST.as_str(),
                 str::replace(&ctx, ":", "_"),
-                KUBECONFIG.to_string()
+                *KUBECONFIG
```

Comment:
I have no idea what the * here does but seems to work!

Reply:
It's the dereferencing operator, usually used to retrieve concrete values from references, i.e. turn a `&i64` into `i64`. It's available on all types implementing [Deref](https://doc.rust-lang.org/core/ops/trait.Deref.html) .
lazy_static stores our lazy string behind a proxy struct `pub struct Lazy<T: Sync>(Cell<Option<T>>, Once);` that implements `Deref` to return the underlying value (`T` in the type signature, which is `String` in our case). So we are dereferencing the proxy struct to get the concrete String value.
Though I'd be lying if I say I really understand its implementation :smile:
This thread tries to explain it: https://users.rust-lang.org/t/after-more-than-half-years-of-rust-in-production-i-still-dont-understand-how-lazy-static-works-what-am-i-doing-wrong/31870